packages/LUIS/package.json

```diff
@@ -8,16 +8,17 @@
     "luis": "./bin/luis.js"
   },
   "directories": {
-    "bin": "bin",
-    "doc": "docs",
     "lib": "lib",
-    "typings": "typings",
-    "examples": "examples"
+    "doc": "docs",
+    "test": "test",
+    "example": "examples",
+    "bin": "bin"
   },
   "files": [
     "bin/",
     "docs/",
     "lib/",
+    "swagger/",
```

Comment:
> "swagger/", [](start = 3, length = 12)

actually, this doesn't need to be in the package.  You had it .npmignored which wasn't right. It should be in the repo but not the package.